### DISCUSSION TITLE
IKEA Remote E1810 gets stuck after the 13th click

### DISCUSSION THREAD

**Discussion by ceysa75** [Koenkk/zigbee2mqtt#5980] [Devices]
Hi all,

I have two E1810 Tradfri Remotes. I made a strange observation that after the 13th click (this is true for all buttons) the device gets stuck i.e. the 14th click is not registered. Only waiting 2 or 3 seconds it again reacts. The Firmware version is 2.3.014. This behaviour is reproducable (happens always) and happens for both remotes. I wonder if this is a shortcoming of the remotes or is this somehow related to zigbee2mqtt. Any help would be appreciated.

Regards
Ceysa

**Comment by Koenkk** [comment]
Known issue in the button firmware: https://www.youtube.com/watch?v=LK9Llve8Xl8&ab_channel=KoenKanters

**Reply by ceysa75** [reply]
Thanks for your response. So, the ball is with IKEA... Let's wait and see.

**Reply by MattWestb** [reply]
All IKEA controlling device is made as light controllers and is sending commands to gropes of light.
The underlying IEEE network is limiting broadcasts to 8 broadcast every 9 seconds and and if trying sending more often all new commands is ignored of the network as designed and also all network router discovery is made with broad casts.

If like have 25 fast clicking switches you shall going for one HA switch that is using unicast and and not broadcast (group sending) but in the end 13 sat clicks is not realistic in the real life.
